frontend-too/src/components/ChatMessages.js

```diff
@@ -1,33 +1,16 @@
-import React, { useEffect } from 'react';
-import { Card, Row, Container } from 'react-bootstrap';
-import { useQuery, gql } from '@apollo/client';
+import React, { useEffect, useState } from 'react';
+import { Card, Row, Container, Form } from 'react-bootstrap';
+import { useQuery } from '@apollo/client';
+import { MESSAGES_SUBSCRIPTION, CHAT_MESSAGES } from '../graphql';
 
-const MESSAGES_SUBSCRIPTION = gql`
-  subscription Subscription {
-    messageSent {
-      id
-      from
-      message
-    }
-  }
-`;
 const ChatMessages = () => {
-  const { subscribeToMore, loading, error, data } = useQuery(gql`
-    {
-      chats {
-        id
-        from
-        message
-      }
-    }
-  `);
+  const { subscribeToMore, loading, error, data } = useQuery(CHAT_MESSAGES);
```

Comment:
useQuery has a callback function called onSuccess that you could use to set the data to chat (state). 

That way you could cut down on some repeat code that you have in your JSX. 

It also has an onError callback that could be utilized if need be.